test/run-tests

```diff
@@ -755,6 +779,35 @@ def check_environment():
         sys.exit(1)
 
 
+def setup_logging(config_logging):
+    log_cfg_default = {
+        "format": "%(asctime)s: %(levelname)s: %(message)s",
+        "datefmt": "%Y-%m-%dT%H:%M:%S",
```

Comment:
IMHO it would be better to mention the UTC offset in the time format to make it easier to compare logs from different sources.